### PR TITLE
fix `Proxy is not able to format code which contains the 'super' expression` (close #1390)

### LINKED ISSUE #1390: Proxy is not able to format code which contains the 'super' expression
Example:
```js
class x extends y {
    method() {
        return super[a];
    }
}
```

Related with https://github.com/DevExpress/testcafe-hammerhead/issues/1389

### PR DESCRIPTION
**Pull request by LavrovArtem**

### CODE REVIEW

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

File: test/server/script-processor-test.js [hunk 1]
```diff
@@ -943,5 +943,12 @@ describe('Script processor', () => {
                 expected: 'async function foo() {  return __get$(obj, "src"); }'
             });
         });
+
+        it('Should process the `super` expression (GH-1390)', () => {
+            testProcessing({
```
Comment (miherlosev) [inline test/server/script-processor-test.js]: Do we cover the `super.prop` expression?

Comment (LavrovArtem) [reply test/server/script-processor-test.js]: Yes

Comment (miherlosev) [reply test/server/script-processor-test.js]: Ok

File: test/server/script-processor-test.js [hunk 2]
```diff
@@ -943,5 +943,12 @@ describe('Script processor', () => {
                 expected: 'async function foo() {  return __get$(obj, "src"); }'
             });
         });
+
+        it('Should process the `super` expression (GH-1390)', () => {
```
Comment (miherlosev) [inline test/server/script-processor-test.js]: Let's rename to `Should process expression with `super` keyword (GH-1390)`

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Pull request merged**